launchd/io.github.lima-vm.socket_vmnet.bridged.en0.plist

```diff
@@ -24,5 +24,7 @@
 		<true />
 		<key>UserName</key>
 		<string>root</string>
+		<key>ProcessType</key>
+		<string>Interactive</string>
```

Comment:
Could you add a comment line to explain this?

Reply:
Sure, will add xml comment above that key, it does look confusing.